routes/arrivals.js

```diff
@@ -147,8 +147,284 @@ Works like \`/stops/{id}/departures\`, except that it uses [\`hafasClient.arriva
 						content: {
 							'application/json': {
 								schema: {
-									type: 'array',
-									items: {type: 'object'}, // todo
+									'type': 'array',
```

Comment:
What do you think about moving the each "root" schema into a variable, to reduce the nesting level?